components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -90,7 +90,7 @@ export default class SchoolCard extends Component {
           <div className={styles.badgeGroup}>
             <Badge
               label={<LabelWithScreenReader label="Online" isActive={props.hasOnline} />}
-              icon={<OnlineIcon />}
+              icon={<OnlineIcon data-testid="OnlineSchools" />}
```

Comment:
what if, instead, you made the testid more specific and dynamic?

```
data-testid={props.hasOnline ? 'School has online' : 'School has no online'}
```

Reply:
Oh, great idea! Will update